content/en/ecosystem/registry/_index.md

```diff
@@ -29,6 +29,12 @@ weight: 20
 
 {{% /blocks/lead %}}
 
+<div style="text-align: center; margin-top: 20px;">
+<button id="scrollToTopBtn" class="scroll-to-top-btn">
```

Comment:
```suggestion
<button id="scrollToTopBtn" class="scroll-to-top-btn">
```

Can you check if you can build the button with any of the predefined buttons from bootstrap? https://getbootstrap.com/docs/5.3/components/buttons/

Reply:
On it.